test/integration/custom-networking/custom_networking_suite_test.go

```diff
@@ -133,6 +134,9 @@ var _ = BeforeSuite(func() {
 		err = f.K8sResourceManagers.CustomResourceManager().CreateResource(eniConfig)
 		Expect(err).ToNot(HaveOccurred())
 	}
+	By("CHANGE IMAGE AND CRD HERE")
+	//at this point each node should have eni's attached from v1alpha1
+	time.Sleep(300 * time.Second)
```

Comment:
this will be removed when this is converted from a draft, just adding a note here do we do not forget